dotnet/src/SemanticKernel.Abstractions/Data/RecordDefinition/VectorStoreRecordDataProperty.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// A description of a data property on a record for storage in a vector store.
```

Comment:
Can we improve this e.g. Defines attributes of a data property on a record which will influence how it is treated by a vector store.

Reply:
Thanks, reworded.